plugins/plugin_compile/project_compile.py

```diff
@@ -933,8 +959,6 @@ def build(self, platform):
         if platform == 'mac':
             script_resource_path = os.path.join(self.app_path, 'Contents/Resources/src')
 
-        self.compile_script(script_resource_path, platform)
```

Comment:
`luacompile` is configured in CMake, running during `cmake --build ~ --config Release`. 

It is not needed  in `cocos compile` procedure.